esp/services/ws_topology/ws_topologyService.cpp

```diff
@@ -1397,7 +1397,11 @@ bool CWsTopologyEx::onTpClusterInfo(IEspContext &context, IEspTpClusterInfoReque
         Owned<IRemoteConnection> conn = querySDS().connect("/Status/Servers/", myProcessSession(),RTM_SUB,SDS_LOCK_TIMEOUT);
         if (conn)
         {
+#ifndef _CONTAINERIZED
             Owned<IConstWUClusterInfo> clusterInfo = getTargetClusterInfo(req.getName());
+#else
+            Owned<IConstWUClusterInfo> clusterInfo = getContainerWUClusterInfoByName(req.getName());
+#endif
```

Comment:
@wangkx - minor, but as with comment I just made here : https://github.com/hpcc-systems/HPCC-Platform/pull/14490/files#r563647433

would be a slightly cleaner if this conditional code was moved into a common method, so each caller didn't have to have the conditional code, e.g. a 'getTargetInfo' in TpWrapper

Reply:
in ws_workunitHelpers there is:
```
#ifndef _CONTAINERIZED
        Owned<IEnvironmentFactory> envFactory = getEnvironmentFactory(true);
        Owned<IConstEnvironment> constEnv = envFactory->openEnvironment();
        Owned<IPropertyTree> root = &constEnv->getPTree();

        Owned<IConstWUClusterInfo> clusterInfo = getTargetClusterInfo(clusterName.str());
#else
        Owned<IConstWUClusterInfo> clusterInfo = getContainerWUClusterInfoByName(clusterName.str());
#endif
```

although as per other comment, doesn't look like environment lines are used (and def. not by getTargetClusterInfo).

It also doesn't check null response of getTargetClusterInfo, where as the above code does.

As per https://github.com/hpcc-systems/HPCC-Platform/pull/14491/files#r563650525 comment, it would be good to common up and put the #ifdef CONTAINERIZED conditional in a wrapper method, e.g. in TpWapper.